src/components/city-table.js

```diff
@@ -0,0 +1,54 @@
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from 'react-sparklines';
+
+const CityTable = (props) => {
+
+  const dataAvg = (array) => {
+    const total = array.reduce((a, b) => a + b, 0);
+    return Math.round(total/array.length);
+  }
+
+  if (props.searches.length > 0) {
+
+  return props.searches.map((search, i) => (
+  
+    <table className="table table-hover" key={i}>
+      <tbody>
+        <tr className="align-middle">
+          <td className="text-center">
+            {search.data.city.name}
+          </td>
+
+          <td>
+          <Sparklines data={search.data.list.map((block)=>{return block.main.temp;})}>
+            <SparklinesLine color="orange" />
+            <SparklinesReferenceLine type="avg" />
+          </Sparklines>
+          <div className="text-center">{dataAvg(search.data.list.map((block)=>{return block.main.temp;}))} {'\xB0F'}</div>
+          </td>
```

Comment:
This section could be more DRY (don't repeat yourself) as you're essentially printing the same code three times